docs/source/quickstart.rst

```diff
@@ -1,22 +1,256 @@
+    "5", "6", "", "", "", "SP006", "1.416666667", "Cleft ego", "Positive"
+    "6", "7", "8", "9", "Male", "SP007", "1.416666667", "Cleft ego", "Positive"
+    "7", "8", "", "", "", "SP008A", "1814.375", "Cleft ego", "Positive"
+    "7", "8", "", "", "", "SP008B", "1814.375", "Cleft ego", "Positive"
+    "8", "9", "", "", "", "SP009A", "0.208333333", "Cleft ego", "Negative"
+    "8", "9", "", "", "", "SP009B", "0.208333333", "Cleft ego", "Negative"
+    "0", "1", "2", "3", "Female", "SP001A", "0.166666667", "Cleft id", "Negative"
+    "0", "1", "2", "3", "Female", "SP001B", "0.166666667", "Cleft id", "Negative"
+    "1", "2", "", "", "", "SP002A", "18.125", "Cleft id", "Negative"
+    "1", "2", "", "", "", "SP002B", "18.125", "Cleft id", "Negative"
+    "2", "3", "", "", "", "SP003A", "1.416666667", "Cleft id", "Negative"
+    "2", "3", "", "", "", "SP003B", "1.416666667", "Cleft id", "Negative"
+    "3", "4", "5", "6", "Male", "SP004A", "0.166666667", "Cleft id", "Positive"
+    "3", "4", "5", "6", "Male", "SP004B", "0.166666667", "Cleft id", "Positive"
+    "4", "5", "", "", "", "SP005A", "14.375", "Cleft id", "Positive"
+    "4", "5", "", "", "", "SP005B", "14.375", "Cleft id", "Positive"
+    "5", "6", "", "", "", "SP006", "1.416666667", "Cleft id", "Positive"
+    "6", "7", "8", "9", "Male", "SP007", "1.416666667", "Cleft id", "Negative"
+    "7", "8", "", "", "", "SP008A", "1814.375", "Cleft id", "Positive"
+    "7", "8", "", "", "", "SP008B", "1814.375", "Cleft id", "Positive"
+    "8", "9", "", "", "", "SP009A", "0.208333333", "Cleft id", "Positive"
+    "8", "9", "", "", "", "SP009B", "0.208333333", "Cleft id", "Positive"
+    "0", "1", "2", "3", "Female", "SP001A", "0.166666667", "Extra eardrum", "Negative"
+    "0", "1", "2", "3", "Female", "SP001B", "0.166666667", "Extra eardrum", "Negative"
+    "1", "2", "", "", "", "SP002A", "18.125", "Extra eardrum", "Negative"
+    "1", "2", "", "", "", "SP002B", "18.125", "Extra eardrum", "Negative"
+    "2", "3", "", "", "", "SP003A", "1.416666667", "Extra eardrum", "Negative"
+    "2", "3", "", "", "", "SP003B", "1.416666667", "Extra eardrum", "Negative"
+    "3", "4", "5", "6", "Male", "SP004A", "0.166666667", "Extra eardrum", "Negative"
+    "3", "4", "5", "6", "Male", "SP004B", "0.166666667", "Extra eardrum", "Negative"
+    "4", "5", "", "", "", "SP005A", "1.416666667", "Extra eardrum", "Negative"
+    "4", "5", "", "", "", "SP005B", "1.416666667", "Extra eardrum", "Negative"
+    "5", "6", "", "", "", "SP006", "1814.375", "Extra eardrum", "Negative"
+    "6", "7", "8", "9", "Male", "SP007", "0.208333333", "Extra eardrum", "Positive"
+    "7", "8", "", "", "", "SP008A", "2.166666667", "Extra eardrum", "Positive"
+    "7", "8", "", "", "", "SP008B", "2.166666667", "Extra eardrum", "Positive"
+    "8", "9", "", "", "", "SP009A", "1.041666667", "Extra eardrum", "Positive"
+    "8", "9", "", "", "", "SP009B", "1.041666667", "Extra eardrum", "Positive"
+
```

Comment:
Can you add a section for the transform stage as well? Provide a 1-2 line explanation of what the transform module does (joins extracted output tables together) and then display the transform module and instruct the user to uncomment the merge code.